app/org/sagebionetworks/bridge/dynamodb/DynamoScheduledActivityDao.java

```diff
@@ -57,48 +52,30 @@ public ScheduledActivity getActivity(String healthCode, String guid) {
         if (dbActivity == null) {
             throw new EntityNotFoundException(ScheduledActivity.class);
         }
+        dbActivity.setTimeZone(timeZone);
```

Comment:
Do we have a unit test to test setting the time zone? Otherwise, how do we know that it works? If this is in a pre-existing test, how did that test pass before this change was made?